benchmark/benchmark_performance/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+#****************************************************************************
+#
+#                           Klepsydra Core Modules
+#              Copyright (C) 2019-2020  Klepsydra Technologies GmbH
+#                            All Rights Reserved.
+#
+#  This file is subject to the terms and conditions defined in
+#  file 'LICENSE.md', which is part of this source code package.
+#
+#  NOTICE:  All information contained herein is, and remains the property of Klepsydra
+#  Technologies GmbH and its suppliers, if any. The intellectual and technical concepts
+#  contained herein are proprietary to Klepsydra Technologies GmbH and its suppliers and
+#  may be covered by Swiss and Foreign Patents, patents in process, and are protected by
+#  trade secret or copyright law. Dissemination of this information or reproduction of
+#  this material is strictly forbidden unless prior written permission is obtained from
+#  Klepsydra Technologies GmbH.
+#
+#****************************************************************************
+
+SET(PROJ_NAME kpsr_benchmark_core_perf)
+PROJECT(${PROJ_NAME})
+ADD_SUBDIRECTORY(smart_pool)
```

Comment:
put the correct name: `event_emitter`

Reply:
Done. Erased files due to tests imported from kpsr-core.